spec/spec_helper.rb

```diff
@@ -1,6 +1,5 @@
 require 'simplecov'
-require 'simplecov-gem-profile'
-SimpleCov.start "gem"
+SimpleCov.start unless SimpleCov.running
```

Comment:
is this one really needed ?

Reply:
yeah it will be once I get to ruby 3.1.  there is some weird issue where the the spec helper gets run twice (no clue why) and Ruby raises an exception as the coverage is already started.